src/config/authInfoConfig.ts

```diff
@@ -15,16 +15,38 @@ import { ConfigFile } from './configFile';
  * const authInfo = await AuthInfoConfig.create(AuthInfoConfig.getOptions(username));
  * ```
  */
-export class AuthInfoConfig extends ConfigFile<ConfigFile.Options> {
+export class AuthInfoConfig extends ConfigFile<AuthInfoConfig.Options> {
   /**
    * Gets the config options for a given org ID.
    * @param username The username for the org.
    */
-  public static getOptions(username: string): ConfigFile.Options {
+  public static getOptions(username: string): AuthInfoConfig.Options {
     return {
       isGlobal: true, // Only allow global auth files
       isState: true,
       filename: `${username}.json`
     };
   }
+
+  /**
+   * Init method.
+   *
+   * **Throws** *{@link SfdxError}{ name: 'NamedOrgNotFound' }* If the username.json file is not found when
+   * options.throwOnNotFound is true or undefined.
+   */
+  public async init(): Promise<void> {
+    return super.init(this.options.throwOnNotFound === undefined ? true : this.options.throwOnNotFound);
+  }
+}
+
+export namespace AuthInfoConfig {
+  /**
+   * Options for the AuthInfoConfig.
+   */
+  export interface Options extends ConfigFile.Options {
+    /**
+     * Indicates if init should throw if the corresponding config file is not found.
+     */
+    throwOnNotFound?: boolean;
```

Comment:
Since `ConfigFile.read` accepts `throwOnNotFound`, couldn't we just add `throwOnNotFound?` to the base `ConfigFile.Options`?  Then we could drop it as an `AuthInfo.init` param and any `ConfigFile` subtype could take advantage of it.